src/main/kotlin/app/hashers/CommitCrawler.kt

```diff
@@ -156,6 +175,26 @@ object CommitCrawler {
                 catch (e: Exception) { null }
                 stream != null && !RawText.isBinary(stream)
             }
+            .filter { diff ->
+                val filePath =
+                    if (diff.getNewPath() != DiffEntry.DEV_NULL) {
+                        diff.getNewPath()
+                    } else {
+                        diff.getOldPath()
+                    }
+
+                // Update ignored paths list. The config file has retroactive
+                // force, i.e. if it was added at this commit, then we presume
+                // it is applied to all commits, preceding this commit.
+                if (diff.getOldPath() == CONF_FILE_PATH) {
+                    ignoredPaths =
+                        getIgnoredPaths(repo, diff.getOldId().toObjectId())
+                }
+
+                !ignoredPaths.any { path ->
+                    filePath.startsWith(path)
```

Comment:
Conf contains "/test/a", file path "/test/apple.txt" and it will be filtered too. But we should have only filter /test/a file or dir.

Reply:
right, will fix it